### PR TITLE
Update 2.1.1

### PR DESCRIPTION
**Pull request by stephen-hoover**
Update to most recent release and update maintainer list.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

^ this is handy! hi!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ source:
     - setup.py.patch
 
 build:
-  number: 1002
+  number: 1000
```
Comment (beckermr) [inline recipe/meta.yaml]: This can be set back to 0.

Comment (beckermr) [reply recipe/meta.yaml]: 1000 is only for same versions with new vs old compilers.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -48,4 +48,4 @@ about:
 extra:
   recipe-maintainers:
     - jseabold
-    - beckermr
```
Comment (beckermr) [inline recipe/meta.yaml]: Please leave me on here.

File: recipe/meta.yaml (hunk 1 above)
Comment (stephen-hoover) [reply recipe/meta.yaml]: Hi Matt! Do you know why it was 1002?

Comment (stephen-hoover) [reply recipe/meta.yaml]: Ah, I see. Thanks.

Comment (beckermr) [reply recipe/meta.yaml]: This was an autogenerated build number when the compilers were being migrated in all of conda-forge. For packages at the same version, the ones built with new compilers were set to 1000 + X so they would be higher priority in the solver. Once a version is bumped, then that always has higher priority over a higher build number of a previous version, so we don't need that anymore.

**Comment by beckermr** [comment]
LGTM! Failing azure builds, if any, can be ignored. That is still experimental. Happy to merge once the other builds pass!

**Comment by beckermr** [comment]
Hmmmm. You will have to look into this locally.

```
Created temporary directory: /tmp/pip-req-build-of1rif
  Added file://$SRC_DIR to build tracker '/tmp/pip-req-tracker-yCSCMB'
    Running setup.py (path:/tmp/pip-req-build-of1rif/setup.py) egg_info for package from file://$SRC_DIR
    Running command python setup.py egg_info
    running egg_info
    running build_src
    build_src
    building extension "_glmnet" sources
      target build/src.linux-x86_64-2.7/glmnet/_glmnetmodule.c does not exist:
       Assuming _glmnetmodule.c was generated with "build_src --inplace" command.
       Yes! Using 'glmnet/_glmnetmodule.c' as up-to-date target.
    f2py options: []
      adding 'build/src.linux-x86_64-2.7/fortranobject.c' to sources.
      adding 'build/src.linux-x86_64-2.7' to include_dirs.
    error: f2py target_c file 'build/src.linux-x86_64-2.7/fortranobject.c' not found
```

There is a patch to the setup.py in this build. You may need to adjust that. I am not 100% sure what the problem is.

**Comment by stephen-hoover** [comment]
Yes, I'm looking into it. I don't think it's a conda-forge issue. It looks like something with the packaging when I uploaded to PyPI.

**Review by beckermr — APPROVED** [review]

**Comment by stephen-hoover** [comment]
Thanks! @beckermr , is the Travis failure a problem? It failed on the Python 3.7 build. It looks like the package built and tested successfully, but there was a final failure,
```
Traceback (most recent call last):

  File "/Users/travis/miniconda3/bin/upload_package", line 11, in <module>

    sys.exit(upload_package())

  File "/Users/travis/miniconda3/lib/python3.7/site-packages/click/core.py", line 764, in __call__

    return self.main(*args, **kwargs)

  File "/Users/travis/miniconda3/lib/python3.7/site-packages/click/core.py", line 717, in main

    rv = self.invoke(ctx)

  File "/Users/travis/miniconda3/lib/python3.7/site-packages/click/core.py", line 956, in invoke

    return ctx.invoke(self.callback, **ctx.params)

  File "/Users/travis/miniconda3/lib/python3.7/site-packages/click/core.py", line 555, in invoke

    return callback(*args, **kwargs)

  File "/Users/travis/miniconda3/lib/python3.7/site-packages/conda_forge_ci_setup/build_utils.py", line 82, in upload_package

    upload_or_check(recipe_root, owner, channel, [config_file])

  File "/Users/travis/miniconda3/lib/python3.7/site-packages/conda_forge_ci_setup/upload_or_check_non_existence.py", line 134, in upload_or_check

    new_distributions = [path for m, path in built_distributions

  File "/Users/travis/miniconda3/lib/python3.7/site-packages/conda_forge_ci_setup/upload_or_check_non_existence.py", line 135, in <listcomp>

    if not built_distribution_already_exists(cli, m, path, owner)]

  File "/Users/travis/miniconda3/lib/python3.7/site-packages/conda_forge_ci_setup/upload_or_check_non_existence.py", line 41, in built_distribution_already_exists

    distro_name)

  File "/Users/travis/miniconda3/lib/python3.7/site-packages/binstar_client/__init__.py", line 442, in distribution

    self._check_response(res)

  File "/Users/travis/miniconda3/lib/python3.7/site-packages/binstar_client/__init__.py", line 230, in _check_response

    raise ErrCls(msg, res.status_code)

binstar_client.errors.ServerError: ('Gateway Timeout: The gateway server did not receive a timely response ([GET] https://api.anaconda.org/dist/conda-forge/glmnet/2.1.1/osx-64/glmnet-2.1.1-py37h436c29b_0.tar.bz2 -> 504)', 504)

travis_time:end:026cb7b5:start=1552353004000000000,finish=1552353120000000000,duration=116000000000
[0K[31;1mThe command "upload_package ./ ./recipe ./.ci_support/${CONFIG}.yaml" exited with 1.[0m
```

**Comment by beckermr** [comment]
I think this error is fine. I've rekicked the CI to be sure.

**Comment by beckermr** [comment]
We might have a bit of trouble with the CI builds on master. github appears to be a bit flakey right now...

**Comment by stephen-hoover** [comment]
@beckermr , it looks like the v2.1.1 binaries are built and available for both Linux and OS X now. Thank you for your help!

**Pull request merged**